packages/map-template/src/components/LocationDetails/LocationDetails.jsx

```diff
@@ -216,16 +180,61 @@ function LocationDetails({ onBack, onStartWayfinding, onSetSize, snapPointSwiped
     }
 
     /**
```

Comment:
```suggestion
    /*
```

It's not a definition (where a JSDoc block should be used), but a function call (where a multiline comment can be used to explain why we call it)

Reply:
It has been fixed 👍 , I didn't know they are different 🤔